workflow_scripts/install_linux_proto.sh

```diff
@@ -1,15 +1,24 @@
 yum -y install wget
-yum -y install ccache
 
-export NUMCORES=`grep -c ^processor /proc/cpuinfo`
+wget http://ftp.mirrorservice.org/sites/sourceware.org/pub/gcc/releases/gcc-5.2.0/gcc-5.2.0.tar.gz
```

Comment:
No, you shouldn't change gcc. If the default one in the manylinux docker image doesn't work, it must because you did something wrong.

Reply:
Actually the purpose of installing gcc here is to get the latest `libstdc++.so.6`.
While installing 3.11.3 protobuf, it encounters the following error:
```
2020-12-31T04:58:13.8924662Z  You could try using --skip-broken to work around the problem
2020-12-31T04:58:13.8925459Z            Requires: libstdc++.so.6(GLIBCXX_3.4.20)(64bit)
2020-12-31T04:58:13.8926555Z Error: Package: protobuf-3.11.2-2.el8.x86_64 (/protobuf-3.11.2-2.el8.x86_64)
2020-12-31T04:58:13.8927400Z            Requires: libstdc++.so.6(CXXABI_1.3.8)(64bit)
2020-12-31T04:58:13.8928777Z Error: Package: protobuf-compiler-3.11.2-2.el8.x86_64 (/protobuf-compiler-3.11.2-2.el8.x86_64)
2020-12-31T04:58:13.8929927Z            Requires: libstdc++.so.6(GLIBCXX_3.4.21)(64bit)
2020-12-31T04:58:13.8931307Z Error: Package: protobuf-compiler-3.11.2-2.el8.x86_64 (/protobuf-compiler-3.11.2-2.el8.x86_64)
2020-12-31T04:58:13.8932580Z            Requires: libstdc++.so.6(CXXABI_1.3.9)(64bit)
```
The original old libstdc++.so.6 does not have such things so I thought I need to get a newer libstdc++.so.6. To do that in CentOS, I only know I can get it from building gcc... `yum install` certain version of libstdC++ will encounter some issues like recursive dependencies failure. Do you have any suggestion for it? Thank you.

Reply:
The link you have is for CentOS 8, but the docker image is based on CentOS 6, so it definitely won't work.
see:
cbs.centos.org/kojifiles/packages/protobuf/3.11.2/2.el8/x86_64/protobuf-3.11.2-2.**el8**.x86_64.rpm

Why don't you just compile protobuf from source? It would be much easier than build gcc.

Reply:
Building protobuf from source is a good idea. Thank you @snnn. Happy New Year!!